gulpfile.js

```diff
@@ -104,7 +106,7 @@ gulp.task("yarn:lint", () => {
 gulp.task("unit:test", () => {
   return gulp
     .src(TYPESCRIPT_SOURCE_DIR)
-    .pipe(run("jest --coverage --runInBand"));
+    .pipe(run("jest --coverage --runInBand --forceExit"));
```

Comment:
whis is `forceExit` needed?

Reply:
because for an unknow reason jest tests won't exit (at least on my win32 machine) when run through gulp (running the cli directly with node_modules/jest/bin/jest.js just works).
it could be a problem with gulp + run plugin + jest (pretty hard to debug, so I've just added the flag and it works)

Reply:
then:

``` js
"jest --coverage --runInBand" + (process.platform === "win32" ? "--forceExit" : "")
```

+ add your comment above as explanation